commercial/app/views/books/book.scala.html

```diff
@@ -1,5 +1,5 @@
-@(book: model.commercial.Book, omnitureId: Option[String], clickMacro: Option[String])(implicit request: RequestHeader)
-@import views.support.{ImgSrc, Item300}
+@import _root_.commercial.model.merchandise.Book
```

Comment:
Whenever I try to use `import commercial.model.merchandise.Book` I am informed by the compiler that there is an ambiguous reference to `commercial`. I believe that this is because. on compilation, all templates are compiled with [default imports](https://github.com/guardian/frontend/blob/master/project/Prototypes.scala#L75). I think that there may be a subpackage of `commercial` within either the `model` package or the `views.support` package.

It is immensely frustrating. Perhaps I am missing something but the only way to directly reference the commercial package I want was to add the `_root_` prefix.